quantus/metrics/robustness_metrics/relative_stability.py

```diff
@@ -0,0 +1,312 @@
+from typing import Union, Dict, Callable, Tuple
+import numpy as np
+import jax.numpy as jnp
+import jax
+from abc import abstractmethod, ABC
+import functools
+from tqdm import tqdm
+
+from ..base import Metric
+from ...helpers import utils, perturb_func
+
+"""
+For example usage, refer to tutorials/tutorial_getting_started_with_tensorflow.ipynb
+"""
+
+
+@functools.partial(jax.jit, static_argnums=(4, 5))
+def relative_stability_objective_single_point(
+    x, x_s, a, a_s, eps_min, division_in_denominator
+):
+
+    nominator = (a - a_s) / (a + (a == 0) * eps_min)  # prevent division by 0
+    nominator = jnp.linalg.norm(nominator)
+
+    denominator = x - x_s
+    if division_in_denominator:
+        denominator /= x + (x == 0) * eps_min
+
+    denominator = jnp.linalg.norm(denominator)
+    denominator = jnp.max(jnp.stack([denominator, eps_min]))
+
+    return nominator / denominator
+
+
+relative_stability_objective_batched = jax.vmap(
+    relative_stability_objective_single_point, in_axes=(0, 0, 0, 0, None, None)
+)
+
+relative_stability_objective_vectorized_over_perturbation_axis = jax.vmap(
+    relative_stability_objective_batched, in_axes=(None, 0, None, 0, None, None)
+)
+
+
+class RelativeStability(Metric, ABC):
```

Comment:
Inherit from Metric (only)

Reply:
without explicitly specifying ABC in inheritance list, user will be able to instantiate `quantus.RelativeStability`
class, which is not intended.

Reply:
there's a test case for this https://github.com/aaarrti/Quantus/blob/relative-stability/tests/metrics/test_relative_stability.py#L43

Reply:
why not instantiate `quantus.RelativeStability`?